hr_user_ciam_update/models/ciam_update.py

```diff
@@ -20,7 +20,7 @@ def action_update_ciam(self):
                 'lastName': self.employee_id.lastname,
                 'eMail': self.employee_id.user_id.email,
                 'username': self.employee_id.user_id.login,
-                'password': self.employee_id.user_id.password if self.employee_id.user_id.password else "Acctest09",
+                #'password': self.employee_id.user_id.password if self.employee_id.user_id.password else "Acctest09",
```

Comment:
Why not just remove it? If you want to leave commented code, please state why above the the code.